app/assets/stylesheets/application.css.scss

```diff
@@ -22,11 +22,17 @@
   */
 
 @import 'foundation-icons';
-
 .background-image {
-  background: image-url('background_picture.jpg')
+  background: image-url('background_picture.jpg');
+  padding: 195px 5px 195px 5px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `195px 5px` instead of `195px 5px 195px 5px`